### PR TITLE
Highlight Active Separators even if leftmost/rightmost

### PR DESCRIPTION
**Pull request by jordwalke**

### CODE REVIEW

File: autoload/wintabs.vim [hunk 1]
```diff
@@ -558,15 +557,7 @@ endfunction
 
 " delete buffer from buflist if it isn't attached to any wintab
 function! s:purge(buffer)
```
Comment (zefei) [inline autoload/wintabs.vim]: let's remove the changes on this file

Comment (jordwalke) [reply autoload/wintabs.vim]: Whoops, didn't mean to include these. Thanks.

File: autoload/wintabs.vim [hunk 2]
```diff
@@ -475,7 +475,6 @@ function! s:switch_tab(n, confirm)
   " set nohidden to trigger confirm behavior
   let hidden = &hidden
   let &hidden = 0
-
```
Comment (zefei) [inline autoload/wintabs.vim]: this as well

**Comment by zefei** [comment]
This is manually merged. Thanks!

**Pull request closed without merging**